//
// This file is part of the WoW++ project.
// 
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
// GNU General Public License for more details.
// 
// You should have received a copy of the GNU General Public License
// along with this program; if not, write to the Free Software 
// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
//
// World of Warcraft, and all World of Warcraft or Warcraft art, images,
// and lore are copyrighted by Blizzard Entertainment, Inc.
// 

syntax = "proto2";
package wowpp.proto;

message TalentEntry {
	required uint32 id = 1;
	required uint32 tab = 2;
	required uint32 row = 3;
	required uint32 column = 4;
	repeated uint32 ranks = 5;	// Spell-IDs
	optional uint32 dependson = 6;
	optional uint32 dependsonrank = 7;
	optional uint32 dependsonspell = 8;
}

message Talents {
	repeated TalentEntry talents = 1;
}
